### PR TITLE
Addressed issue #268

### PR DESCRIPTION
**Pull request by felixbohnacker**
I removed the public modifiers from test classes and test cases.

### CODE REVIEW

**Comment by stevesap** [comment]
`[ERROR] /home/circleci/cwa-server/common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyTest.java:[68,9] method testSubmissionTimestampGetter() is already defined in class app.coronawarn.server.common.persistence.domain.DiagnosisKeyTest`

**Comment by felixbohnacker** [comment]
Sorry, I must have made a stupid mistake there 😕 I removed the duplicate method.

**Comment by stevesap** [comment]
No worries @felixbohnacker . Thanks for your help!

File: common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilderTest.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 
-public class DiagnosisKeyBuilderTest {
+ class DiagnosisKeyBuilderTest {
```
Comment (stevesap) [inline common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilderTest.java:37]: Hi @felixbohnacker, could you please remove the leading space here before `class` and also everywhere else below where a leading space was introduced? Thanks!

Comment (felixbohnacker) [reply common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilderTest.java]: Thank you for the remark, I removed them.

**Pull request merged**